sagemaker-jumpstart/object_detection/JumpStart_Object_Detection.ipynb

```diff
@@ -0,0 +1,796 @@
+{
```

Comment:
<p>Recommended to use code font formatting using backticks for these parameters, ie <code>deploy_image_uri</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>done</p>